common/lib/xmodule/xmodule/modulestore/search.py

```diff
@@ -10,7 +10,7 @@
 LOGGER = getLogger(__name__)
 
 
-def path_to_location(modulestore, usage_key, full_path=False):
+def path_to_location(modulestore, usage_key, request, full_path=False):
```

Comment:
Do you really need the whole request object here?

Reply:
as I am getting **masquerade_settings** and **request_user**, I used the request object instead of sending two different arguments.